docs/user-guide/global-configurations/catalog-framework.md

```diff
@@ -0,0 +1,87 @@
+# Catalog Framework
+
+## Introduction [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+Ideally, all resources such as microservices, clusters, jobs, pods, etc. should contain detailed information so that its users know what each of those resources do, how to use them, as well as all their technical specs. Access to such data makes it easier for engineers to quickly discover and understand the relevant resources.
+
+To achieve this, Devtron supports a feature known as **Catalog Framework**. Using this, you as a [super-admin](../global-configurations/authorization/user-access.md#role-based-access-levels) can decide the data you expect from the managers of different resource types. In other words, you can create a custom JSON schema that would ultimately render a form for the resource owners to fill. Once the form is filled, a GUI output will appear as shown below.
+
+![Sample Catalog Data for an App](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/catalog-framework/sample-app-catalog.jpg)
+
+Currently, Devtron supports catalog framework for the following resource types (a.k.a. resource kind):
+
+* [Devtron applications](../../reference/glossary.md#devtron-apps)
+* [Helm applications](../../reference/glossary.md#helm-apps)
+* [Clusters](../../reference/glossary.md#cluster)
+* [Jobs](../../reference/glossary.md#job)
+
+There are two parts involved in the creation of a desirable resource catalog:
+
+1. [Defining a Schema](#defining-a-schema) 
+2. [Filling the Schema-Generated Form](#filling-the-schema-generated-form)
+
+---
+
+## Defining a Schema
+
+Only a super-admin can create/edit a schema.
```

Comment:
Start give some visual seperation for RBAC, such that a repeating user may just know in there muscle memory where to look for RBAC related information

Reply:
Good idea! Will start incorporating a hintblock and title it as "Who Can Perform This Action?" (a layman phrase).